src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
```

Comment:
2. Error messages:
When throwing an exception, consider providing a more informative error message. The message in the `RuntimeException` constructor should be meaningful to users or developers who encounter the error. This will help in troubleshooting and understanding the cause of the error.